test/function.localstack.test.ts

```diff
@@ -430,6 +431,37 @@ localstackTestSuite("functionStack", (testResource, _stack, _app) => {
     })
   );
 
+  test(
+    "Call Lambda CloudWatch SDK",
+    (parent) => {
+      const alarm = new aws_cloudwatch.Alarm(parent, "Alarm", {
+        evaluationPeriods: 1,
+        threshold: 1,
+        metric: new aws_cloudwatch.Metric({
+          namespace: "AWS/Lambda",
+          metricName: "Errors",
+        }),
+      });
+
+      return new Function(
+        parent,
+        "function",
+        localstackClientConfig,
+        async () => {
+          const result = await $AWS.SDK.CloudWatch.describeAlarms({
+            params: {
+              AlarmNames: [alarm.alarmName],
+            },
+            iamResources: [alarm.alarmArn],
```

Comment:
Hmm, how does it know which IAM Action to authorize against the Resource?

Reply:
That's what I use the iam service name mapping for: I get the service prefix from there and then just use the method name. E.g. 

- `$AWS.SDK.CloudWatch.describeAlarms => "cloudwatch:DescribeAlarms"`
- `$AWS.SDK.StepFunctions.describeExecution => "states:DescribeExecution"`